### DISCUSSION TITLE
공지사항

### DISCUSSION THREAD

**Discussion by tjdgoz** [code-bootcamp/odisca_client#35] [공지사항]
## 👋 Welcome!
📢 오디스카 팀의 원활한 소통을 위해 디스커션 창구를 만들었습니다!! 공지사항은 이쪽에 올릴 예정이니 자주 확인 부탁드려요 :)
  
 간단한 폴더 사용설명
  * 머지현황: 오늘 머지 현황을 알릴 폴더에요.
  * 업무분배: 대략적인 기능 구현이 끝나면 업무분배가 어떻게 되어 작업을 했는지 정리해놓을 폴더에요.
  * 일정관리: 기능구현이 완료된 API와 완료되지 않은 API를 확인할 수 있고 오늘의 업무분배를 확인할 수 있어요.
  * 제보하기: **pull 받은 작업물에 에러나 수정해야할 사항이 있을경우 이곳에 꼭 공유해주세요!!** 
  * 프론트엔드 규칙: 우리가 지켜야할 코드 규칙들이나, 오디스카 UI상 지켜주어야 할 사항들을 확인할 수 있고, 규칙관련해서 자유롭게 글 올려주세요

제가 임의로 정한 폴더들이라서, 추가하면 좋을 것 같은 폴더나 의견 있으면 댓글 남겨주세요!! 오늘도 화이뗑 💚

**Comment by jiiyaeya** [comment]
good idea
